src/enums.jl

```diff
@@ -0,0 +1,24 @@
+@enum Polarisation begin
+    VV
+    VH
+    HV
+    HH
+end
+
+function Base.parse(::Type{Polarisation}, polarisation::AbstractString)
+    
+    polarisation = uppercase(polarisation)
+    
+    if polarisation == "VV"
+        return VV
+    elseif polarisation == "VH"
+        return VH
+    elseif polarisation == "HV"
+        return HV
+    elseif polarisation == "HH"
+        return HH
+    else
+        throw(ErrorException("$polarisation is not a valid polarisation"))
```

Comment:
I would maybe change error message to something like: "$polarisation is not valid. Only single-pol: VV, VH, HV or HH are implemented for now"